src/test/java/edu/cornell/kfs/module/purap/document/service/impl/CuCreditMemoServiceImplTest.java

```diff
@@ -62,28 +60,26 @@
     private PurapService purapService;
     private VendorService vendorService;
     private CuVendorCreditMemoDocument creditMemoDocument;
-    @Mock
     private static Person mo14Person;
-    @Mock
     private static UserSession mo14Session;
 
     @Before
 	public void setUp() throws Exception {
-        accountsPayableService = EasyMock.createNiceMock(AccountsPayableService.class);
-        EasyMock.expect(accountsPayableService.getExpiredOrClosedAccountList(creditMemoDocument)).andReturn(new HashMap<>());
-        EasyMock.replay(accountsPayableService);
-
-        dataDictionaryService = EasyMock.createNiceMock(DataDictionaryService.class);
-        EasyMock.expect(dataDictionaryService.getAttributeMaxLength(DocumentHeader.class, KRADPropertyConstants.DOCUMENT_DESCRIPTION)).andReturn(200);
-        EasyMock.replay(dataDictionaryService);
-
+        accountsPayableService = Mockito.mock(AccountsPayableService.class);
+        Mockito.when(accountsPayableService.getExpiredOrClosedAccountList(creditMemoDocument)).thenReturn(new HashMap<>());
```

Comment:
It looks like creditMemoDocument is null at this point, and I think the old test was the same way. Should this be modified to just check for any argument? Or could this be placed after the document has been set up?

Reply:
I changed it to Mockito.any